src/components/image/ImageWithCaption.tsx

```diff
@@ -0,0 +1,67 @@
+import { FC } from "react";
+import Image from "next/image";
+
+type ImageWithCaption = {
+  image: string;
+  width: number;
+  height: number;
+  altText: string;
+  caption?: string;
+  attribution?: string;
+  alignment: "left" | "center" | "right";
+};
+
+const ImagesWithCaption: FC<ImageWithCaption> = ({
+  image,
+  width,
+  height,
+  altText,
+  caption,
+  attribution,
+  alignment,
+}) => {
+  let flexAlignment = getFlexAlignment(alignment);
+  let textAlignment = getTextAlignment(alignment);
+
+  return (
+    <div className={`flex ${flexAlignment}`}>
+      <div
+        style={{
+          width: width,
+        }}
+      >
+        <div className="relative">
+          <Image src={image} alt={altText} height={height} width={width} />
```

Comment:
**question:** Would we want to pass thru sizes & styles like the above component does?